lib/home/users/permissions.dart

```diff
@@ -5,7 +5,7 @@ import 'package:thc/utils/theme.dart';
 import 'package:thc/utils/widgets/error_dialog.dart';
 
 class Permissions extends StatelessWidget {
-  const Permissions({Key? key, required this.user}) : super(key: key);
+  const Permissions({super.key, required this.user});
```

Comment:
We needed to downgrade the `lints` package in order to use `agora_uikit`. I removed the package for now, and if we re-add in the future, hopefully it won't be an issue, thanks to [a PR I submitted there](https://github.com/AgoraIO-Community/VideoUIKit-Flutter/pull/188).

With the new linter rules, I just needed to hit "save", and it tidied up this constructor!